### PR TITLE
Add openssh 7.3 info for agent forwarding and fix issue 15

### PR DESCRIPTION
**Pull request by gdestuynder**

### CODE REVIEW

**Comment by gdestuynder** [comment]
@claudijd or @ameihm0912 if you have a few minutes to review/potentialyl merge this :)

File: Security/Guidelines/OpenSSH.mediawiki [hunk 1]
```diff
@@ -364,7 +380,33 @@ These are not officially supported and may require you to recompile OpenSSH.
 * [https://github.com/devinteske/apple/tree/master/OpenSSH-186/openssh OpenSSH MacOS X] (see also http://devinteske.com/)
 }}
 
-=== Safer alternative to SSH agent forwarding ===
+=== Recommended, safer alternatives to SSH agent forwarding ===
+
+==== OpenSSH >=7.3  ====
+
+OpenSSH 7.3 onwards allow users to jump through several hosts in a rather automated fashion. It has fully support for
```
Comment (claudijd) [inline Security/Guidelines/OpenSSH.mediawiki]: s/fully/full/

File: Security/Guidelines/OpenSSH.mediawiki [hunk 2]
```diff
@@ -364,7 +380,33 @@ These are not officially supported and may require you to recompile OpenSSH.
...
+
+For example to reach an host being a bastion/jumphost:
+<source code="bash">
+# Single jump
+$ ssh -J ssh.mozilla.com myhost.private.scl3.mozilla.com
+
+# Jump through 2 hops
+$ ssh -J ssh.mozilla.com,ec2-instance.aws.net 10.0.0.3
+
+# Copy data from an host
```
Comment (claudijd) [inline Security/Guidelines/OpenSSH.mediawiki]: s/an/a/

File: Security/Guidelines/OpenSSH.mediawiki [hunk 3]
```diff
@@ -364,7 +380,33 @@ These are not officially supported and may require you to recompile OpenSSH.
...
+
+OpenSSH 7.3 onwards allow users to jump through several hosts in a rather automated fashion. It has fully support for
+scp and sftp commands as well as regular ssh.
+
+For example to reach an host being a bastion/jumphost:
+<source code="bash">
+# Single jump
+$ ssh -J ssh.mozilla.com myhost.private.scl3.mozilla.com
+
+# Jump through 2 hops
```
Comment (claudijd) [inline Security/Guidelines/OpenSSH.mediawiki]: This comment in markdown is being interpreted as a numbered list and I don't think that is what you are going for here.

**Comment by claudijd** [comment]
@gdestuynder very sorry for the delay here, I saw the original mention but forgot to circle back until you mentioned in #infosec. Just a couple minor changes, but overall r+

File: Security/Guidelines/OpenSSH.mediawiki (hunk 3 above)
Comment (gdestuynder) [inline Security/Guidelines/OpenSSH.mediawiki]: its mediawiki syntax not markdown :)

**Comment by claudijd** [comment]
r+

**Pull request merged**